pkg/collector/py/kubeutil.c

```diff
@@ -0,0 +1,21 @@
+// +build cpython,kubelet
```

Comment:
Can you add the LICENSE disclaimer? (Not sure why it's missing from all the C code)